### PR TITLE
Add Electron recipe

### PR DESCRIPTION
**Pull request by auchenberg**
Closing #10

### CODE REVIEW

File: Electron/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Electron debugging (main and renderer process)
+
+by [Kenneth Auchenberg](https://twitter.com/auchenberg)
+
+This recipe shows how to use the built-in Node Debugger and the [Debugger for Chrome](https://github.com/Microsoft/vscode-chrome-debug) extension with VS Code to debug [Electron](https://electron.atom.io) applications.
+
+[Electron](https://electron.atom.io) consists of two proceses: A Main process running in NodeJS and a render process in Chromium. This means that you'll need to use two debugger instances within VS Code to debug both processes. This is the reason for why you both the built-in Node Debugger and the [Debugger for Chrome](https://github.com/Microsoft/vscode-chrome-debug).
```
Comment (roblourens) [inline Electron/README.md]: Not necessarily, you can have no renderer process or multiple renderer processes, or additional processes spawned by the renderer.

File: Electron/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+  ```json
+    {
+        "version": "0.2.0",
+        "configurations": [
+            {
+                "type": "node",
+                "request": "launch",
+                "name": "Electron: Main",
+                "protocol": "inspector",
```
Comment (roblourens) [inline Electron/README.md]: Worth mentioning somewhere that this depends on the Electron version? I suppose in this tutorial they'll get the latest one

File: Electron/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    }
+  ```
+
+  ## Debugging of the main process
+  
+  1. Go ahead and set a breakpoint in **main.js** on `line 16` within the `createWindow` function.
+
+  2. Go to the Debug view, select the **'Electron: Main'** configuration, then press F5 or click the green play button.
+
+  3. VS Code should now attempt to start your Electron app, and your breakpoint on `line 16` in `main.js` should be git
```
Comment (roblourens) [inline Electron/README.md]: Should be git?

File: Electron/README.md (hunk 2 above)
Comment (auchenberg) [reply Electron/README.md]: Feedback incorporated.

**Pull request merged**